data-esnext.js

```diff
@@ -91,6 +91,14 @@ exports.tests = [
       name: 'basic support',
       mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SIMD',
       exec: function () {/*
+        simdFloatTypes=['Float32x4'];
```

Comment:
Great.  The instructions on the home page says to prefer the most compatible form when coding tests, I believe this is it.  But github is indicating that you still expect changes, did I miss something?